libs/java/service/src/main/java/ca/bc/gov/tno/services/kafka/config/KafkaConsumerConfig.java

```diff
@@ -0,0 +1,112 @@
+package ca.bc.gov.tno.services.kafka.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * KafkaConsumerConfig class, provides configuration settings for the Kafka
+ * Consumer.
+ */
+@Configuration
+public class KafkaConsumerConfig extends KafkaConfig {
```

Comment:
I'd love to know how to make optional @Configuration beans in java.  Presently I have to make all @Value properties optional so that they do not throwing errors in services that don't need them.